fe/fe-core/src/main/java/com/starrocks/alter/SchemaChangeHandler.java

```diff
@@ -1340,6 +1342,98 @@ public void updateTableMeta(Database db, String tableName, Map<String, String> p
         }
     }
 
+
+    public void updateBinlogConfigMeta(Database db, Long tableId, Map<String, String> properties,
+                                       BinlogManager.BinlogContext context, TTabletMetaType metaType) throws DdlException {
+        List<Partition> partitions = Lists.newArrayList();
+        OlapTable olapTable;
+        BinlogConfig newBinlogConfig;
+        boolean hasChanged = false;
+
+        db.readLock();
+        try {
+            olapTable = (OlapTable) db.getTable(tableId);
+            partitions.addAll(olapTable.getPartitions());
+            if (olapTable.isBinlogConfigNull()) {
+                newBinlogConfig = new BinlogConfig();
+                hasChanged = true;
+            } else {
+                newBinlogConfig = new BinlogConfig(olapTable.getCurBinlogConfig());
+            }
+        } finally {
+            db.readUnlock();
```

Comment:
After obtaining the db write lock, it will check whether there are concurrent modifications through binlog version, and if so, the binlog config will be remodified